<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbIngresar">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="LabelRegistrarse" name="lRegistrarse">
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lClave">
     <property name="text">
      <string>Clave</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lUsuario">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="leUsuario"/>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="leClave">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelRegistrarse</class>
   <extends>QLabel</extends>
   <header location="global">labelregistrarse.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leUsuario</tabstop>
  <tabstop>leClave</tabstop>
  <tabstop>pbIngresar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
